sqrt_function_algorithm/sqrt_cplusplus.cpp

```diff
@@ -0,0 +1,32 @@
+#include<iostream.h>
+#include<conio.h>
+
+double sq(double n)
+{
+	double beg=0,last=n,mid;
+	while(beg<=last)
+	{
+		mid=(beg+last)/2;
+		if(n>mid*mid)
+		{
+		if(n-mid*mid<=0.000001)return mid;
+		}else
+		{
+		if(mid*mid-n<=0.000001)return mid;
+		}
+		if(n>mid*mid)beg=mid;
+		else last=mid;
+	}
+	return -1;
+}
+void main()
```

Comment:
Also, ```void main()``` is explicitly prohibited by the C++ standard and shouldn't be used. Can you change this to ```int main()```. You can read more about this on [stackoverflow](https://stackoverflow.com/questions/204476/what-should-main-return-in-c-and-c)